src/pages/SearchPage/index.js

```diff
@@ -0,0 +1,183 @@
+import PropTypes from 'prop-types';
```

Comment:
Noting here as well @rushatgabhane @adhorodyski @lukemorawski This page was added but the original SearchPage.js was not removed from the `/pages` folder so the app is actually using code from the original file

Reply:
@mountiny Actually I did remove the original file but probably it slipped in when merging this branch with `main`.